docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -80,6 +80,28 @@ CodeChecker, you just have to run the CodeChecker analyze command with the
 You can read more about Z3 Theorem Prover
 [here](https://github.com/Z3Prover/z3/wiki).
 
+## Z3 SMT Solver
```

Comment:
I'm not sure we should just copy the summary from the report to here.
A few lines about the feature should be enough if someone wants to read in more detail link to the detailed documentation is there.